pkg/loader/bundle/bundle.go

```diff
@@ -170,24 +171,24 @@ func loadPorts(service Service) ([]kobject.Ports, string) {
 			Protocol:      p,
 		})
 	}
-	return ports, ""
+	return ports, nil
 }
 
 // LoadFile loads dab file into KomposeObject
-func (b *Bundle) LoadFile(files []string) kobject.KomposeObject {
+func (b *Bundle) LoadFile(files []string) (kobject.KomposeObject, error) {
 	komposeObject := kobject.KomposeObject{
 		ServiceConfigs: make(map[string]kobject.ServiceConfig),
 		LoadedFrom:     "bundle",
 	}
 	file := files[0]
 	buf, err := ioutil.ReadFile(file)
 	if err != nil {
-		log.Fatalf("Failed to read bundles file: %s ", err)
+		return kobject.KomposeObject{}, errors.Wrap(err, "ioutil.ReadFile failed, Failed to read bundles file")
```

Comment:
@surajssd about the good practice part, it's a very subjective opinion, but we can use it to track down the error when the error messages will get printed in app.go

and about the the error message part, I added the name of the failing functions wrt the function they were failing in, so it gets easier to track down the error.